native/src/main/java/org/ballerinalang/redis/ConnectionCommands.java

```diff
@@ -97,4 +97,23 @@ public static Object close(BObject redisClient) {
             return createBError(e);
         }
     }
+
+    /**
+     * Retrieve information and statistics about the cluster observed by the current node. This command is
+     * exclusively available in cluster mode.
+     * If the connection is in a non-clustered mode, the API will return a `redis:Error`. Other errors will also be
+     * appropriately handled.
+     *
+     * @param redisClient redis client BObject
+     * @return a bulk-string-reply as a string array or, a `redis:Error` if the connection is non-clustered or
+     * encounters any other exceptions.
+     */
+    public static Object clusterInfo(BObject redisClient) {
```

Comment:
Since this functionality is only there for Redis cluster-client, shall we move this into a separate java class ?

Reply:
+1. Moved with https://github.com/ballerina-platform/module-ballerinax-redis/pull/200/commits/795bc934617bff1e97c1c37fcd5557385dab64ea